### DISCUSSION TITLE
How to take screenshot of sddm login screen?

### DISCUSSION THREAD

**Discussion by tryprncp** [prasanthrangan/hyprdots#1983] [Q&A]
I see some tutorials on some forums on how to take screenshot of sddm login screen but it's specific for xorg. do you guys have a workaround on how to take a screenshot of sddm login screen? i dont want to recreate my setup in vm yet

**Comment by kRHYME7** [comment]
Capture it using another PC using video capture. 


another is simulate sddm using test mode

**Comment by se7en-x230** [comment]
sleep 5 ; grimblast copy
